tests/integration/04.Edge.Tests.ps1

```diff
@@ -89,6 +89,40 @@ Describe "Edge" {
         disconnect-nsxserver
     }
 
+    Context "Edge Status" {
+
+        BeforeAll{
+            if ( -not ( Get-NsxEdge $name ) ) {
+                New-NsxEdge -Name $name -Interface $vnics[0],$vnics[1],$vnics[2] -Cluster $cl -Datastore $ds -password $password -tenant $tenant -enablessh -Hostname "pestertest"
+            }
+        }
+
+        It "Get Edge Status" {
+            $status = Get-NsxEdge $name | Get-NsxEdgeStatus
+            $status | should not be $null
+            $status.systemStatus| should be "good"
+            $status.edgeStatus | should be "GREEN"
+            $status.publishStatus| should be "APPLIED"
+        }
+
+        It "Get Edge Service Status" {
+            $service = Get-NsxEdge $name | Get-NsxEdgeStatus
+            $service | should not be $null
+
+            ($service.featureStatuses.featureStatus | where-object { $_.service -eq 'firewall' }).status | should be "Applied"
```

Comment:
Would prefer not to test each individual service status, as this list of services is driven by the features of the edge/platform and could change between versions.

maybe make it more generic, and test for the existence of $service.featureStatuses.featureStatus and I think that would suffice for this test. 

And then create another test to test an individual feature status, but just choose the first feature that is listed and make sure it has a service and status parameter.